doc/user/scanners.xml

```diff
@@ -146,15 +146,16 @@ over the file scanning rather than being called for each input line:
 
   <para>
 
-    &SCons; has built-in scanners that know how to look in
+    &SCons; has routines that know how to look in
```

Comment:
I'd leave scanner.

Reply:
This was an attempt to step around the plural problem without having gone to the trouble of actually addressing it, which I'll get to someday.

Reply:
Hint: don't bother using the github phone app to ever *do* anything with an issue, it's just too limited, and the opposite of straightforward.

Reply:
This has been updated.